.github/workflows/run-acceptance-tests.yml

```diff
@@ -42,13 +42,17 @@ jobs:
         uses: ./.github/actions/build
 
   test:
+    needs:
```

Comment:
I noticed that the build/test was failing, but the integration tests were still running. I think it's better to fail fast here.